client/package.json

```diff
@@ -11,6 +11,9 @@
     "react-scripts": "1.1.4",
     "redux": "^4.0.0"
   },
+  "devDependencies": {
```

Comment:
Locally it worked for me installing redux-dev-tools as a client dev dependency but in order to have it deploy to Heroku I had to include it in the root package.json